lib/node_modules/@stdlib/string/remove-first/docs/types/index.d.ts

```diff
@@ -51,7 +114,6 @@
 */
 declare function removeFirst( str: string, n?: number ): string;
 
-
 // EXPORTS //
```

Comment:
```suggestion

// EXPORTS //
```

I believe we want to keep this line break in order to have 2 empty lines between section headers.